devel/Dockerfile

```diff
@@ -35,6 +38,21 @@ RUN git clone --recursive https://github.com/phoenix-rtos/phoenix-rtos-project.g
     cd .. && rm -rf phoenix-rtos-project && \
     chmod -R a+rwX /opt/phoenix/utils
 
+
+#install qemu from xilinx
+RUN git clone https://github.com/Xilinx/qemu.git && \
+    cd qemu && \
+    git checkout tags/xilinx_v2021.2 && \
+    git submodule update --init dtc && \
+    mkdir build && cd build && \
+    ../configure --target-list="aarch64-softmmu" --enable-fdt --disable-kvm --disable-xen --enable-gcrypt && \
+    make -j4 && \
+    make install && \
```

Comment:
huh.... that actually installs everything under `/usr/local`, so why are we copying some binaries to `/opt/xilinx`?

Do we need all of this installed (and others - like desktop shortcuts and icons for qemu?)
![Screenshot_20211109_171428](https://user-images.githubusercontent.com/3939983/140961986-1b547502-7208-4d9f-bfc2-4036f388808c.png)

Reply:
I copied the created binary into `/usr/bin`. 
The qemu from xilinx reuse specific files from `/usr/shares/qemu`, which have been installed within basic `qemu` installation. The total size of the docker increase of 20 MB.

Reply:
nice! I like the improvement over 400 additional MB previously :+1: